dts/arm/st/g0/stm32g071X6.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This one is not known neither

Reply:
It is listed in the Keil pack, but that could be an error. Should I remove it?

Reply:
I have removed it now as I only found this in the keil dfp and not on the ST website.